srcCpp/RTIDDSImpl.cxx

```diff
@@ -3842,6 +3842,317 @@ double RTIDDSImpl_FlatData<T>::obtain_dds_deserialize_time_cost_override(
 }
 #endif
 
+void PerftestDDSPrinter::initialize(ParameterManager *_PM)
+{
+    PerftestPrinter::initialize(_PM);
+    this->_PM = _PM;
+    topicName = std::string("PerftestInfo");
+}
+
+void PerftestDDSPrinter::initialize_dds_entities()
+{
+    DDS_ReturnCode_t retcode;
+
+    DDS_DomainParticipantFactoryQos factory_qos;
+    DDS_DomainParticipantQos dpQos;
+    DDS_PublisherQos publisherQos;
+    DDS_DataWriterQos dwQos;
+
+  #ifndef RTI_MICRO
```

Comment:
Can we check what is the behavior when using micro? We might need to make this differently.